civihr_employee_portal/civihr_employee_portal.install

```diff
@@ -433,6 +433,18 @@ function civihr_employee_portal_update_7033() {
   TaxonomyHelper::createDefaultTerms();
 }
 
+/**
+ * Revert updated features components
+ * - Panel page which holds the My Details page
+ * - Output Filter to write pure HTML in views sections without filters
+ * Refresh the node files as onboarding form node was altered
+ */
+function civihr_employee_portal_update_7034() {
+  features_revert(['civihr_employee_portal_features' => ['page_manager_pages']]);
+  features_revert(['civihr_employee_portal_features' => ['filter']]);
+  civicrm_initialize();
```

Comment:
why do we need this in the update?

Reply:
To create a page for My Details page and to create a filter used in views. It is in the comments of the function.

Reply:
Sure.

Reply:
I already ping @deb1990 in slack about this.

Reply:
Resolved.